src/utilities/theme/tests/utils.test.ts

```diff
@@ -24,6 +25,18 @@ describe('setTextColor', () => {
   });
 });
 
+describe('setBorderColor', () => {
+  it('sets a css variable to white if the variant is dark', () => {
+    const textColor = setBorderColor('topBar', 'dark');
+    expect(textColor).toStrictEqual(['topBar', tokens.colorSkyDark]);
+  });
+
+  it('sets a css variable to inkLighter if the variant is light', () => {
+    const textColor = setBorderColor('topBar', 'light');
+    expect(textColor).toStrictEqual(['topBar', tokens.colorInkLighter]);
+  });
+});
+
```

Comment:
Adding more tests to bump coverage
https://media.tenor.com/images/618f1cc8de509138d44a37bfd5e3412f/tenor.gif